src/runtime/pkg/katautils/config.go

```diff
@@ -295,6 +296,24 @@ func (h hypervisor) firmware() (string, error) {
 	return ResolvePath(p)
 }
 
+func (h hypervisor) snpCertsPath() (string, error) {
+	p := h.SnpCertsPath
+
+	if p == "" {
+		return "", nil
+	}
```

Comment:
Should we fall back to the default when this path is empty? Will this condition be triggered when the config entry is not set at all?

Reply:
I kept it just to handle if the path is not specified in the config file. Yes, this condition will be triggered if the config entry is not set at all. By default, a default path will be specified in the config file and user can change that path.

Reply:
To me the default path means the path that will be used if nothing is specified in the file. I think we should end up with the empty path only if the default path and the path in the config both point to files that don't exist.

Reply:
> To me the default path means the path that will be used if nothing is specified in the file. I think we should end up with the empty path only if the default path and the path in the config both point to files that don't exist.

Not really.  Think about the case of any other configuration file.  Those will not have this value set by default.

Reply:
This arg only does anything when SNP is enabled and when the certs file exists so I think the other cases are fine.

Reply:
@niteeshkd @fitzthum I confess I got confused with this new config option. Reading the code I understood that the file is mandatory. am I right?

If `snp_certs_path` is mandatory and I got not file on host, will it crash Kata?

Reply:
The config is optional. If it is default/not set, then it is not an error if the file doesn't exist. This is good because not everybody cares about attestation so not everyone will have configured the certs.

If the path is set but the certs don't exist at that location, then there will be an error.

Reply:
got it. Being a little slow these days... thanks @fitzthum !